src/test/java/orm/dsl/dml/DMLQueryBuilderDeleteTest.java

```diff
@@ -0,0 +1,77 @@
+package orm.dsl.dml;
+
+import config.PluggableH2test;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+import persistence.sql.ddl.Person;
+import persistence.sql.ddl.mapper.PersonRowMapper;
+
+import java.util.List;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static orm.util.ConditionUtils.eq;
+import static steps.Steps.Person_엔티티_생성;
+import static steps.Steps.테이블_생성;
+
+public class DMLQueryBuilderDeleteTest extends PluggableH2test {
+
+    @Test
+    @DisplayName("DELETE 절 생성 테스트")
+    void DML_DELETE_문_테스트() {
+        // given
+        DMLQueryBuilder dmlQueryBuilder = new DMLQueryBuilder();
+
+        // when
+        String query = dmlQueryBuilder.deleteFrom(Person.class)
+                .build();
+
+        // then
+        assertThat(query).isEqualTo("DELETE FROM person");
+    }
+
+    @Test
+    @DisplayName("DELETE 절 조건 포함 실행 테스트")
+    void DML_DELETE_문_조건절_테스트() {
+        // given
+        DMLQueryBuilder dmlQueryBuilder = new DMLQueryBuilder();
+
+        // when
+        String query = dmlQueryBuilder.deleteFrom(Person.class)
+                .where(
+                    eq("id", 1L)
+                        .and(eq("name", "설동민"))
+                        .or(eq("age", 30))
+                )
+                .build();
```

Comment:
```suggestion
        String query = dmlQueryBuilder.deleteFrom(Person.class)
                .where(
                    eq("id", 1L)
                        .and(eq("name", "설동민"))
                        .or(eq("age", 30))
                )
                .build();
```
와... 대단해요 감탄이 나오네요 👍 👍 👍